packages/app/src/components/admin/AdminPage.tsx

```diff
@@ -25,6 +22,24 @@ const tabs = [
 ];
 
 export const AdminPage = () => {
+  const [tabs, setTabs] = useState([...initialTabs]);
+  const { loading: loadingPermission, allowed: canDisplayRBACTab } =
+    usePermission({
+      permission: policyEntityReadPermission,
+      resourceRef: policyEntityReadPermission.resourceType,
+    });
+  useEffect(() => {
+    if (canDisplayRBACTab && !loadingPermission) {
+      setTabs(prevTabs => [
+        ...prevTabs,
```

Comment:
Could you expand this after the RBAC tab below?  That way this tab shows up first.